.changeset/olive-dryers-nail.md

```diff
@@ -0,0 +1,9 @@
+---
+'@twilio-paste/box': minor
+'@twilio-paste/text': minor
+'@twilio-paste/core': minor
+---
+
+[Box] Small updates to usage of `styled` to create the `Box` component. Small type updates (to adopt utility types).
+
+[Text] Update tests to account for `stylis` parser change in jest emotion matchers.
```

Comment:
todo: we should split these up into separate changesets, otherwise the Box change will appear in the Text changelog and vice-versa

Reply:
Ah ok, I thought it was that similar changes should be in the same changeset. Will split. Thanks!